oss_build.sh

```diff
@@ -88,17 +88,41 @@ for python_version in $PYTHON_VERSIONS; do
     bazel clean
   fi
 
-  if [ "$python_version" = "3.6" ]; then
-    export PYTHON_BIN_PATH=/usr/bin/python3.6 && export PYTHON_LIB_PATH=/usr/local/lib/python3.6/dist-packages
-  elif [ "$python_version" = "3.7" ]; then
-    export PYTHON_BIN_PATH=/usr/local/bin/python3.7 && export PYTHON_LIB_PATH=/usr/local/lib/python3.7/dist-packages
-    ABI=cp37
-  elif [ "$python_version" = "3.8" ]; then
-    export PYTHON_BIN_PATH=/usr/bin/python3.8 && export PYTHON_LIB_PATH=/usr/local/lib/python3.8/dist-packages
-    ABI=cp38
+  if [ "$(uname)" = "Darwin" ]; then
+    if [ "$python_version" = "3.6" ]; then
+      export PYTHON_BIN_PATH=python3.6
+    elif [ "$python_version" = "3.7" ]; then
+      export PYTHON_BIN_PATH=python3.7
+      ABI=cp37
+    elif [ "$python_version" = "3.8" ]; then
+      export PYTHON_BIN_PATH=python3.8
+      ABI=cp38
+    else
+      echo "Error unknown --python. Only [3.6|3.7|3.8]"
+      exit
+    fi
+
+    export PYTHON_LIB_PATH=`$PYTHON_BIN_PATH -c 'import site; print("\\n".join(site.getsitepackages()))'`
+
+    bazel_config=""
+    version=`sw_vers -productVersion | sed 's/\./_/g' | cut -d"_" -f1,2`
+    PLATFORM="macosx_${version}_x86_64"
```

Comment:
need to do anything special for M1?

Reply:
I didn't have the M1 environment.

Reply:
I need a support for M1 .... Thanks

Reply:
sorry，I didn't find an easy way to get the M1 env.

Reply:
OK, I use `uname -m`